pkg/controller/defaults/defaults_test.go

```diff
@@ -0,0 +1,24 @@
+package defaults
+
+import (
+	"testing"
+
+	"github.com/acorn-io/acorn/pkg/scheme"
+	"github.com/acorn-io/baaah/pkg/router/tester"
+)
+
+func TestProjectWorkloadClassSwap(t *testing.T) {
+	tester.DefaultTest(t, scheme.Scheme, "testdata/workloadclass/projectWorkloadClassSwap", ProjectWorkloadClass)
```

Comment:
nit: other testdata directories are kebab-case.

Reply:
These files got removed as what they were testing is no longer relevant!